src/components/lists/items/CallListItem.jsx

```diff
@@ -11,7 +11,7 @@ export default class CallListItem extends React.Component {
 
     render() {
         let call = this.props.data;
-        let timestamp = Date.utc.create(call.allocation_time);
+        let timestamp = new Date(call.allocation_time);
```

Comment:
Why this change? This will potentially render a date in the wrong timezone, offsetting it by one or more hours. I just noticed this so I'm commenting now even though I know this is early in the process. I just don't want to forget. Hope you don't mind.